x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/notifications/AnomalyDetectionAuditMessage.java

```diff
@@ -23,6 +23,11 @@ public AnomalyDetectionAuditMessage(String resourceId, String message, Level lev
         super(resourceId, message, level, timestamp, nodeName);
     }
 
+    @Override
+    public final String getJobType() {
+        return "anomaly_detector";
```

Comment:
To make it clearer where this comes from you could use:

```
    return Job.ANOMALY_DETECTOR_JOB_TYPE;
```

Reply:
Done.